### ISSUE TITLE
AppImage does not run with AppImageLauncher

### ISSUE THREAD

**Issue by adazem009** [musescore/MuseScore#18830]
### Issue type

Other type of issue

### Bug description

Latest nightly builds don't work at all on Linux when AppImageLauncher is installed. The AppImage crashes with this error message:
```
execv error: No such file or directory
```
When I try to extract it using `--appimage-extract`:
```
This doesn't look like a squashfs image.
Failed to open squashfs image
```

### Steps to reproduce

1. Download a nightly build of MuseScore or an artifact from GitHub actions.
2. Install AppImageLauncher.
3. Mark the AppImage as executable and execute it.

### MuseScore Version

4.2.0

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Ubuntu 22.04 (KDE Plasma 5.24.7)

### Additional context

I think this PR is related: #18785
My system has both version 2 and 3 of `libfuse` installed.

**Comment by AntonioBL** [comment]
Ciao.
I tried to test this inside a virtual machine with Ubuntu 22.04, KDE 5.92.0 Plasma 5.24.7, but I could not reproduce this behavior: the Appimage was running fine.
Are you launching the Appimage via a launcher, or directly launching it from the command line?

**Comment by adazem009** [comment]
> Ciao.
> I tried to test this inside a virtual machine with Ubuntu 22.04, KDE 5.92.0 Plasma 5.24.7, but I could not reproduce this behavior: the Appimage was running fine.
> Are you launching the Appimage via a launcher, or directly launching it from the command line?

I have AppImageLauncher installed and I clicked "run once" to run the AppImage.

**Comment by AntonioBL** [comment]
Can you please try to launch the AppImage via command line and see if it gives problems?

**Comment by shoogle** [comment]
Looks like we're waiting on https://github.com/TheAssassin/AppImageLauncher/issues/585 to be resolved. In the meantime, it's a choice between:

1. Continue using the new (static) AppImage runtime, in which case AppImageLauncher users needed to stop uninstall AppImageLauncher in order to use our nightly builds.

2. Revert back to using the old (non-static) AppImage runtime, in which case Ubuntu users need to install `libfuse2` to run our nightly builds, which they have probably done already in order to run our stable builds.

On that basis I think we should probably revert back for now. @AntonioBL, would you be willing to do the honours?

**Comment by AntonioBL** [comment]
Ok, done: https://github.com/musescore/MuseScore/pull/18854
It is a pity that we cannot already use the new static runtime...

**Comment by shoogle** [comment]
@AntonioBL, Indeed! Thanks for testing and reverting it for us. Let's hope it gets supported soon in AppImageLauncher.

**Comment by adazem009** [comment]
> @AntonioBL, Indeed! Thanks for testing and reverting it for us. Let's hope it gets supported soon in AppImageLauncher.

Should this issue be closed and #18633 reopened until it gets supported in AppImageLauncher?

**Comment by shoogle** [comment]
@adazem009, good point. The fixes are mutually exclusive, and we've chosen to prioritise this issue for now as explained [here](https://github.com/musescore/MuseScore/issues/18830#issuecomment-1656306321).
